src/pip/_internal/commands/download.py

```diff
@@ -126,14 +125,11 @@ def run(self, options, args):
                     None
                 )
 
-                preparer = RequirementPreparer(
-                    build_dir=directory.path,
-                    src_dir=options.src_dir,
-                    download_dir=options.download_dir,
-                    wheel_download_dir=None,
-                    progress_bar=options.progress_bar,
-                    build_isolation=options.build_isolation,
+                preparer = self.make_requirement_preparer(
+                    temp_directory=directory,
+                    options=options,
                     req_tracker=req_tracker,
+                    download_dir=options.download_dir
```

Comment:
nit: Add trailing commas to multi-line call sites, since that makes diffs easier to read when modifying this stuff.

Reply:
done